src/main/java/hellos/Hellos.java

```diff
@@ -9,12 +9,18 @@ public class Hellos {
   public String generateOutput() {
     StringBuilder builder = new StringBuilder();
 
+    builder.append(jackSaysHello());
     builder.append(kkSaysHello());
     builder.append(nicSaysHello());
 
     return builder.toString();
   }
 
+  private String jackSaysHello() {
+    // TODO Auto-generated method stub
+    return "Jack says 'Hello!'\n";
+  }
```

Comment:
You should remove comments like the `TODO` comments that get generated by the system.

```suggestion
  private String jackSaysHello() {
    return "Jack says 'Hello!'\n";
  }
```